### PR TITLE
Added isCuid() to check if string is a valid 'cuid'

### PR DESCRIPTION
**Pull request by aat2703**

### CODE REVIEW

**Comment by aat2703** [comment]
@endyjasmi

**Comment by endyjasmi** [comment]
Hello @aat2703 , thanks for the PR and sorry for the delay as I'll need to wait until I have free time before reply.

File: src/Cuid.php [hunk 1]
```diff
@@ -231,4 +231,20 @@ public static function slug()
...
+	 * @return boolean
+	 */
+
+	public static function isCuid($cuid)
+	{
+		if (gettype($cuid) !== 'string') {
+			return false;
+		}
+
+		return $cuid[0] === 'c';
```
Comment (endyjasmi) [inline src/Cuid.php]: @aat2703 Do you think you can use RegExp to check the format for the cuid before returning the check result. The cuid itself has a deterministic format.

Comment (aat2703) [reply src/Cuid.php]: Maybe... it's implemented as in the node library for cuid
source: https://github.com/ericelliott/cuid/blob/753a47ae1016719b7175721858590ebc75ca8c29/index.js#L69

Comment (endyjasmi) [reply src/Cuid.php]: Fair enough.

**Pull request merged**